openreview/conference/templates/areachairWebfield.js

```diff
@@ -78,11 +79,17 @@ var loadData = function(result) {
   }
 
   var invitationsP = Webfield.getAll('/invitations', {
-    invitation: WILDCARD_INVITATION, invitee: true,
+    regex: WILDCARD_INVITATION, invitee: true,
```

Comment:
Just out of curiosity, what is the difference between these 2? I know it's possible to do some basic regex queries using the id or invitation params.

Reply:
invitation parameter in the get invitation call doesn't exist(the backend ignores it), it is only supported in the get notes call. I guess this is a bug.

Reply:
Oh right, I didn't notice that. But what about the id param? Could you use that instead of regex?

Reply:
We can use id or regex. 

If we use id, we need to set '.*' in the id parameter.
If we use regex, we can send any regex.

Reply:
Good to know, thanks!